frontend/src/components/validation/SearchForm.js

```diff
@@ -152,16 +158,60 @@ const SearchForm = (props) => {
     if (param === "order") {
       setDoRagnge(false);
     }
-    getFromOpenElisServer("/rest/user-test-sections", fetchTestSections);
+    switch(searchBy){
+      case "routine":
+        let testSectionId = new URLSearchParams(window.location.search).get(
```

Comment:
Since youve used Lexical Declarations inside your case block ,
can you just wrap your case blocks inside curly braces /block statements ie 

```
case "routine":{
        let testSectionId = new URLSearchParams(window.location.search).get(
          "testSectionId"
        );
        testSectionId = testSectionId ? testSectionId : "";
        getFromOpenElisServer("/rest/user-test-sections", (fetchedTestSections) => {
          let testSection = fetchedTestSections.find(testSection => testSection.id === testSectionId);
          let testSectionLabel = testSection ? testSection.value : "";
          setDefaultTestSectionId(testSectionId);
          setDefaultTestSectionLabel(testSectionLabel);
          fetchTestSections(fetchedTestSections);
        })
        if(testSectionId){
          let values = { unitType: testSectionId };
          handleSubmit(values);
        }
        break;
}
      case "order":
```